lib/manageiq/api/common/application_controller_mixins/request_body_validation.rb

```diff
@@ -7,6 +7,7 @@ class BodyParseError < ::RuntimeError
           end
 
           def self.included(other)
+            other.class_attribute :openapi_enabled, :default => true
```

Comment:
LGTM though I'm not sure that this parameter belongs in this particular module?  Not sure what other module we would use though.  @bdunne Thoughts?